data/wanderer/wanderers start.txt

```diff
@@ -1868,9 +1878,10 @@ mission "Wanderers Hai Assistance 2"
 				goto threaten
 			label attack
 			`	"I will not authorize preemptive violence against our own flesh and blood," says one of the elders. "We only defend ourselves against them, never willingly initiate an attack."`
+			`	Teeneep and another Elder who seems to be in her corner both chitter the Hai equivalent of "Hear Hear" to this, though she is being very careful with her contributions.`
 			label threaten
 			`	"Perhaps it would be enough merely to threaten them," says another of the elders. "We could gather a massive war fleet above Cloudfire. The Unfettered would need to muster a matching fleet above Darkcloak. It would draw many of their warships away from the alien territory."`
-			`	After well over an hour of further discussion, they reach consensus that that is their best option. As you leave the meeting, Sayari says, "And now, I would like to visit the Unfettered themselves. Perhaps we can talk sense into them."`
+			`	After well over an hour of further discussion, and with no small backing from Teeneep, they reach the consensus that this is their best option. As you leave the meeting, Sayari exchanges a few words with Teeneep before approaching you, "And now, I would like to visit the Unfettered themselves. Perhaps we can talk sense into them."`
```

Comment:
They should be backing the decision Teeneep suggested of moving a fleet to the border, not doing it over her objections.

Sayari is going to the Unfettered despite Teeneep warning her it is a fool's errand.

Reply:
If you want to take a pass at updating the dialog, please do so. Otherwise I'll work on it after I fix the mission coding bugs.

Reply:
A pass has been taken.